docs/encyclopedia/dataconversion.mdx

```diff
@@ -213,7 +213,7 @@ The Temporal CLI and the Web UI in turn provide built-in hooks to call the Codec
 You can perform some operations on your Workflow Execution using the Temporal CLI and the Web UI.
 For example. you can start or signal an active Workflow Execution from the Temporal CLI or cancel a Workflow Execution from the Web UI, which might require inputs that contain sensitive data.
 
-To encode this data, [specify your Codec Server endpoints](#codec-server) with [the Temporal CLI](/cli/cmd-options#codec-endpoint) and configure your Web UI to use the Codec Server endpoints.
+To encode this data, [specify your Codec Server endpoints](#codec-server) with the `codec-endpoint` parameter in [the Temporal CLI](/cli) and configure your Web UI to use the Codec Server endpoints.
```

Comment:
```suggestion
To encode this data, specify your [Codec Server endpoints](#codec-server) with the `codec-endpoint` parameter in [the Temporal CLI](/cli) and configure your Web UI to use the Codec Server endpoints.
```

This is a fussy suggestion that minimizes the horizontal width of the link. Links that are more specific and localized are more readable.